cmake/modules/RootbenchCTest.cmake

```diff
@@ -0,0 +1,20 @@
+#-------------------------------------------------------------------------------
+#
+#  RootbenchCTest.cmake
```

Comment:
Maybe we should call this RootBenchCTest.cmake?

Reply:
ok! will be done!